test/WebJobs.Script.Tests.Integration/Management/FunctionsSyncManagerTests.cs

```diff
@@ -606,6 +692,116 @@ public async Task TrySyncTriggers_DurableV2ExtensionJson_V2ConfigPosted()
+                var hostConfig = GetHostConfig(durableConfig: null, useBundles: false, version: "2.0", concurrencyConfig: concurrency);
+
+                SetupHttpTrigger(hostConfig.ToString());
+                var syncResult = await _functionsSyncManager.TrySyncTriggersAsync();
+
+                Assert.True(syncResult.Success, "SyncTriggers should return success true");
+                Assert.True(string.IsNullOrEmpty(syncResult.Error), "Error should be null or empty");
+
+                VerifyResultWithSections(concurrencyPayload: GetExpectedConcurrencyPayload(), extensionsPayload: GetExpectedDefaultHttpExtensionPayload());
+            }
+        }
+
+        private JObject GetDurableExtensionsSection()
+        {
+            var durableConfig = new JObject();
+
+            durableConfig["hubName"] = "DurableTask";
+
+            var azureStorageConfig = new JObject();
+            azureStorageConfig["connectionStringName"] = "DurableConnection";
+            durableConfig["storageProvider"] = azureStorageConfig;
+            return durableConfig;
+        }
+
+        private JObject GetConcurrencySection()
+        {
+            var concurrencyConifg = new JObject();
+            concurrencyConifg["dynamicConcurrencyEnabled"] = true;
+            concurrencyConifg["snapshotPersistenceEnabled"] = true;
+            return concurrencyConifg;
+        }
+
+
+
+        private string GetExpectedConcurrencyPayload()
+        {
+            return "{\"dynamicConcurrencyEnabled\":true,\"maximumFunctionConcurrency\":500,\"cpuThreshold\":0.8,\"snapshotPersistenceEnabled\":true}";
+        }
+
+        private string GetDefaultConcurrencyPayload()
+        {
+            return "{\"dynamicConcurrencyEnabled\":false,\"maximumFunctionConcurrency\":500,\"cpuThreshold\":0.8,\"snapshotPersistenceEnabled\":true}";
+        }
+
+        private string GetExpectedExtensionPayload()
+        {
+             return "{\"durableTask\":{\"httpSettings\":{\"defaultAsyncRequestSleepTimeMilliseconds\":30000},\"hubName\":\"DurableTask\",\"storageProvider\":{\"connectionStringName\":\"DurableConnection\"},\"tracing\":{\"traceInputsAndOutputs\":false,\"allowVerboseLinuxTelemetry\":false,\"traceReplayEvents\":false,\"distributedTracingEnabled\":false,\"distributedTracingProtocol\":\"HttpCorrelationProtocol\"},\"notifications\":{\"eventGrid\":null},\"maxConcurrentActivityFunctions\":null,\"maxConcurrentOrchestratorFunctions\":null,\"localRpcEndpointEnabled\":null,\"extendedSessionsEnabled\":false,\"extendedSessionIdleTimeoutInSeconds\":30,\"maxOrchestrationActions\":100000,\"overridableExistingInstanceStates\":1,\"entityMessageReorderWindowInMinutes\":30,\"useGracefulShutdown\":false,\"rollbackEntityOperationsOnExceptions\":true,\"useAppLease\":true,\"appLeaseOptions\":{\"renewInterval\":\"00:00:25\",\"acquireInterval\":\"00:05:00\",\"leaseInterval\":\"00:01:00\"}}}";
```

Comment:
Might make sense for us to have some static test file resources that we load for verification. That'll make it much easier to view/maintain. I.e. add baseline files to a Resources directory that is copied to test output (or embedded in test assembly).

Reply:
That's a good idea. Will do that.

Reply:
Done